frontend/public/components/modals/create-namespace-modal.jsx

```diff
@@ -114,6 +116,23 @@ const CreateNamespaceModalWithTranslation = connect(
         [allow]: t('public~No restrictions'),
         [deny]: t('public~Deny all inbound traffic'),
       };
+
+      const popOverText = () => {
+        const type = this.props.createProject ? t('public~project') : t('public~namespace');
```

Comment:
We're currently capitalizing k8s kinds in the UI, so this would be Project or Namespace.

Reply:
Changed.  Noted in the description that this change deviates from the design documentation.